spring-boot-example/cifuzz.yaml

```diff
@@ -22,9 +22,8 @@
 
 ## Command-line arguments to pass to libFuzzer.
 ## See https://llvm.org/docs/LibFuzzer.html#options
-engine-args:
- - -rss_limit_mb=4096
- - --instrumentation_includes=com.example.**
+#engine-args:
```

Comment:
Why are these options deleted?

Reply:
`rss_limit_mb` is set dynamically by Jazzer based on the available heap size and should not be set manually. 
The new Jazzer version should use the project structure to automatically set the instrumentation filer. Apparently that does not work for this project so I added the flag back in.

Reply:
Thanks! the remote run works.